spec/factories/users.rb

```diff
@@ -14,6 +15,10 @@
       end
     end
 
+    trait :without_image do
+      image nil
+    end
```

Comment:
:+1:  This makes sense. Since it is not reused, another option would be making it an inner factory: 
```ruby
factory: :user_without_image { image nil }
```
and just create `user_without_image` factory